### PR TITLE
making ready for simdem use

### PR DESCRIPTION
**Pull request by SorraTheOrc**
Separate out prep and demo scripts
Provide some scripts to easily run the prep and demo scripts
improvements to the prep script

NOTE
The path to SimDem is currently hard coded in demo.sh and prep.sh - this needs to be resolved. Currently does not work with SimDem in a container which would be a better way of running.

### CODE REVIEW

**Comment by SorraTheOrc** [comment]
@suhaildawood please review

File: incubator/dcos_spark_install/preparation/script.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+```
+RESOURCE_GROUP=acs-spark-demo
+SERVICE_NAME=acs-dcos-spark
+DNS_NAME_PREFIX=acs-dcos-spark
+REGION=eastus
+```
+
+```
+echo region is $REGION
+```
```
Comment (SorraTheOrc) [inline incubator/dcos_spark_install/preparation/script.md:18]: NOTE: this does not work. Environment variables are not carried across from one command execution to the next. It is possible to pass in environment variables to SimDem execution environments but the format is JSON. We need to think about how to make these environment variables work.

Comment (suhaildawood) [reply incubator/dcos_spark_install/preparation/script.md:18]: I've opened up a PR that adds support for persistent environment variables via a `env.json` file (https://github.com/rgardler/simdem/pull/1)

**Comment by suhaildawood** [comment]
Looks good! I've made some local changes to the script that will be easier to add once this PR is merged into master.

**Pull request merged**